### PR TITLE
fix open up spot when connection closes test

### PR DESCRIPTION
**Pull request by aliyakamercan**
@DanSimon

### CODE REVIEW

**Comment by DanSimon** [comment]
👍

File: colossus-tests/src/test/scala/colossus/core/ServerSpec.scala [hunk 1]
```diff
@@ -255,16 +254,9 @@ class ServerSpec extends ColossusSpec {
           withServer(server) {
             val c1 = TestClient(server.system, TEST_PORT)
             expectConnections(server, 1)
-            val c2 = TestClient(server.system, TEST_PORT, waitForConnected = false, connectRetry = NoRetry)
```
Comment (nsauro) [inline colossus-tests/src/test/scala/colossus/core/ServerSpec.scala]: Shouldn't this test still be maintaining that the Server rejects connections when it is at capacity?

Comment (nsauro) [inline colossus-tests/src/test/scala/colossus/core/ServerSpec.scala]: Hrm, maybe not, I just saw the test above this tests for that.

**Comment by nsauro** [comment]
👍

**Comment by DanSimon** [comment]
Can you add a brief description to the PR as well as a link to the issue.

I'm not entirely sure this fixes the test, as the test should have worked before as well, but this at least simplifies it so if it starts having problems again it should be easier to diagnose.

**Pull request merged**